compiler/test/dotty/tools/repl/ShadowingTests.scala

```diff
@@ -97,7 +109,7 @@ class ShadowingTests extends ReplTest(options = ShadowingTests.options):
          |""".stripMargin
   )
 
-  @Test def `shadow subdirectories on classpath` =
+  @Test def shadowSubDir =
```

Comment:
I kind of understand the first form.

Reply:
I didn't want to try to figure out how to glob for spaces.  After testing it now I can't find a way, I can only star it like `-- *shadow*sub*`...